apps/Makefile

```diff
@@ -1,14 +1,20 @@
-include apps/calculation/Makefile
-include apps/graph/Makefile
+include apps/shared/Makefile
 include apps/home/Makefile
-include apps/hardware_test/Makefile
 include apps/on_boarding/Makefile
-include apps/probability/Makefile
-include apps/regression/Makefile
+include apps/hardware_test/Makefile
+snapshots =
+
+# You can select below which apps you want to include in your build, as well as
+# their order on the home screen. Each Makefile below is responsible for setting
+# the $snapshots variable (name of the snapshot class) and the $snapshot_headers
+# (path to the snapshot header).
+include apps/calculation/Makefile
```

Comment:
The list of Makefiles to include should be stored inside a variable and displayed inside `make info` (like APPS_LIST in #73). Otherwise automatized tools like a build-your-own-image couldn't actually select which apps to build and it would needlessly pollute `git status` too.

Reply:
You are right. I am though merging this commit as it is a good step forward and make the necessary change in a future commit.